### DISCUSSION TITLE
What data am I using?

### DISCUSSION THREAD

**Discussion by meganewing** [sr320/course-fish546-2021#37] [Q&A] [answered]
Still unclear as to if I should use Giles/Chris's coho data or Roberts lab data.

Fish preferred but since this is for the sake of learning, I'm not too picky :) 

When looking at [Geoduck Novaseq Files](https://sr320.github.io/Geoduck-NovaSeq-files/ ), I was a little unclear as to what exactly the context of the data was.

**Comment by sr320** [comment, marked as answer]
Good question! 
The geoduck data you would be able to look at differences in expression in different tissues, adults vs juveniles, and how ocean acidification impacts gene expression.

See if you can have them drop the data on ostrich.. using the credentials I provided. Or just have them contact me.
